tests/integration_tests/performance/test_boottime.py

```diff
@@ -30,80 +30,6 @@
 }
 
 
-@pytest.fixture
-def fast_microvm(microvm_factory, guest_kernel_linux_4_14, rootfs_rw):
-    """The microvm defined for the boottime SLA
-
-    Guest kernel 4.14
-    Rootfs: Ubuntu 22.04 ext4
-
-    Using ext4 seems to result in a faster boot than with squashfs. Probably
-    because we have to spend CPU time decompressing and extracting into memory.
-    """
-    return microvm_factory.build(kernel=guest_kernel_linux_4_14, rootfs=rootfs_rw)
-
-
-def test_no_boottime(uvm_plain):
-    """
-    Check that boot timer device is not present by default.
-    """
-    vm = uvm_plain
-    _configure_and_run_vm(vm)
-    # microvm.start() ensures that the vm is in Running mode,
-    # so there is no need to sleep and wait for log message.
-    timestamps = re.findall(TIMESTAMP_LOG_REGEX, uvm_plain.log_data)
-    assert not timestamps
-
-
-def test_boottime_no_network(fast_microvm, record_property, metrics):
-    """
-    Check boot time of microVM without a network device.
-    """
-
-    vm = fast_microvm
-    vm.jailer.extra_args.update({"boot-timer": None})
-    _configure_and_run_vm(vm)
```

Comment:
This function (I can't comment on it because it's outside the diff) is unused now and can also be removed :)